### DISCUSSION TITLE
Overwrite question while opening sqlite db

### DISCUSSION THREAD

**Discussion by radozd** [pawelsalawa/sqlitestudio#4704] [Q&A]
Hi,
I'm trying to add existing database to the app. After selecting file and pressing OK I'm getting this:

<img width="329" alt="image" src="https://user-images.githubusercontent.com/60126193/221393284-a988eef5-581a-47b3-ba9b-4babe0255b3e.png">

**Comment by pawelsalawa** [comment]
I assume you're using SQLiteStudio 3.3.x or older? If yes, you need to use different button for selecting existing file vs selecting inexisting (to be created) file. They are next to each other, in DB dialog.

In SQLiteStudio 3.4.x there is just 1 button for both cases and it accepts both existing and inexisting files.

**Reply by radozd** [reply]
In fact, no. I took the latest release 3.4.3 and it has an issue I mentioned. Two buttons were ok.

**Reply by pawelsalawa** [reply]
Interesting. I will look into it. You can hit "replace". Nothing will be replaced. The selected file is passed to sqlite3_open() function in either way, which opens existing files, or creates them if they don't exist.

If you're hesitating, you can make a copy of the db file. Just to be safe.

**Reply by pawelsalawa** [reply]
Created #4705 to track it.
